### PR TITLE
FIX: export all logs in the log page

### LINKED ISSUE #1884: [Bug]: Export all Logs exports only 150 lines
### Environment

Release

### Bug Description

The option to "Export All Logs" only also only exports the last 150 lines (as the other option does) 

### Launcher Version

2.1.4

### Operating System

Windows

### Affected Service (if applicable)

--

### Logs (if applicable)

```shell
--
```

### Acknowledgment

- [X] I have checked the existing bugs and confirmed this issue is not already reported.

### PR DESCRIPTION
**Pull request by mabasian**
fix #1884

### CODE REVIEW

**Review by NeoPlays — CHANGES_REQUESTED** [review]
I don't think all of this duplicated code is needed

**Review by NeoPlays — COMMENTED** [review]
duplicted code

**Comment by mabasian** [comment]
@NeoPlays The `updateAndExportAllLogs` and `getAllServiceLogs` functions have been updated to ensure they work with a single button click. Please review the changes when you get a chance.

File: launcher/src/components/UI/node-page/NodeScreen.vue [hunk 1]
```diff
@@ -110,8 +112,9 @@ onMounted(() => {
   }, 2000);
 
   updateConnectionStats();
-
+  updateAllServiceLogs();
   updateServiceLogs();
+  pollingLogs = setInterval(updateAllServiceLogs, 10000); // refresh logs
```
[the remark this answers is not in the document]
Comment (mabasian) [reply launcher/src/components/UI/node-page/NodeScreen.vue]: All the intervals removed related to this func

File: launcher/src/backend/Monitoring.js [hunk 2]
```diff
@@ -2935,6 +2935,75 @@ rm -rf diskoutput
...
   }
 
+  // Get all service logs from docker without any tail limit
+  // Accepts an optional object of arguments:
+  // [OPTIONAL] logs_since=<ISO8601DateString>: Get logs since (must be a valid ISO 8601 date string, e.g. 2013-01-02T13:23:37Z)
+  // [OPTIONAL] logs_until=<ISO8601DateString>: Get logs until (must be a valid ISO 8601 date string, e.g. 2013-01-02T13:23:37Z)
+  // [OPTIONAL] logs_ts=<bool>: When true each log line is prefixed with a ISO8601DateString timestamp (default is false)
+  // [OPTIONAL] service_name=<string>: When specified only logs for the specified case-sensitive service name (e.g.: "BesuService") are returned
+  // Returns array of services including their docker logs on success or empty array on errors
+  async getAllServiceLogs(args) {
```
[the remark this answers is not in the document]
Comment (mabasian) [reply launcher/src/backend/Monitoring.js:2945]: Function changed

**Review by NeoPlays — APPROVED** [review]
lgtm 🚀

**Pull request merged**